install/kots/manifests/kots-config.yaml

```diff
@@ -33,6 +33,39 @@ spec:
           help_text: You may either use an in-cluster container registry or configure your own external container registry for better performance. This container registry must be accessible from your Kubernetes cluster.
           recommended: false
 
+        - name: reg_incluster_storage
+          title: In-cluster Storage provider
+          type: select_one
+          when: '{{repl (ConfigOptionEquals "reg_incluster" "1") }}'
+          default: none
+          help_text: You may configure your docker registry to use an external storage backend.
```

Comment:
```suggestion
          help_text: You may configure your Docker registry to use an external storage backend.
```

Reply:
It is worth! but this is still `in-cluster` registry right? Here we are just asking `incluster` registry to use an `S3` backend.

This configuration is passed down to the `registry` deployments ENV, but not replace it 🤔

Reply:
My worry is that the suggested second sentence could be confusing as it is suggesting to use AWS Elastic Container Registry but we are just using S3 right? 🤔

Reply:
As far as my understanding goes, AWS ECR doesn't work with Gitpod because ECR doesn't automatically create a new repo if it doesn't exist when pushing the image. For that reason, if you're using AWS as your platform, I believe that you have to use in-cluster registry with S3 backing (as you've done).

Maybe the sentence should be `This setting is recommended for AWS users instead of using Elastic Container Registry` and then (future PR) link to something in the docs that explains this in more detail.

Reply:
Yep, That makes sense! Updated